backend/presentation/views.py

```diff
@@ -0,0 +1,17 @@
+from rest_framework.viewsets import ModelViewSet
+
+from presentation.models import Presentation, Tag
+from presentation.permissions import IsAdminOrOwner, IsAdminOrCanOnlyGetPost
+from presentation.serializers import PresentationSerializer, TagSerializer
+
+
+class PresentationViewSet(ModelViewSet):
+    queryset = Presentation.objects.all()
+    serializer_class = PresentationSerializer
+    permission_classes = [IsAdminOrOwner]
+
+
+class TagViewSet(ModelViewSet):
+    queryset = Tag.objects.all()
+    serializer_class = TagSerializer
+    permission_classes = [IsAdminOrCanOnlyGetPost]
```

Comment:
Maybe you meant this?
```suggestion
    permission_classes = [IsAdminOrReadOnly]
```

Reply:
User should POST new Tag so `IsAdminOrReadOnly` won't work with this. That's why I created custom permission class and add POST to SAFE_METHODS list.